go/interpreter/lfvm/interpreter_test.go

```diff
@@ -468,23 +456,15 @@ func TestRunGenerateResult(t *testing.T) {
 			expectedErr:    fmt.Errorf("unexpected error in interpreter, unknown status: %v", statusSelfDestructed+1),
 			expectedResult: tosca.Result{},
 		},
-		"getOutput fail": {
-			setup: func(ctx *context) {
-				ctx.resultSize = uint256.Int{1, 1}
-			},
-			expectedResult: tosca.Result{
-				Success: false,
-			},
-		},
 	}
 
 	for name, test := range tests {
 		t.Run(fmt.Sprintf("%v", name), func(t *testing.T) {
 
-			ctxt := getCtxt()
-			if test.setup != nil {
-				test.setup(&ctxt)
-			}
+			ctxt := context{}
+			ctxt.refund = baseRefund
+			ctxt.gas = baseGas
+			ctxt.returnData = bytes.Clone(baseOutput)
```

Comment:
this is a nice detail. I guess it is important to check that a different instance must have the same values.